scss/grid/_layout.scss

```diff
@@ -266,7 +266,9 @@
         }
 
         .k-grid-edit-row td > .k-textbox,
-        .k-edit-cell > .k-textbox {
+        .k-grid-edit-row td > .k-widget,
+        .k-edit-cell > .k-textbox,
+        .k-edit-cell > .k-widget {
```

Comment:
Move this and the following lines down to line 669, where the edit cell is.

Also, create a class for the command cell and put the styles for the button there too.